lib/cc/cli/source_fingerprint.rb

```diff
@@ -0,0 +1,69 @@
+require "digest/md5"
+
+module CC
+  module CLI
+    class SourceFingerprint
+      def initialize(output)
+        @output = output
+      end
+
+      def compute
+        md5 = Digest::MD5.new
+        md5 << key
```

Comment:
Ah okay. I didn't know that we'd considered and discussed the impact of that change.

One downside might be that more PRs will fail. And editing an existing issue - even improving it - will show up as a `new` issue.

But it's hard to have a perfect system so maybe this tradeoff is okay